h/presenters/annotation_json.py

```diff
@@ -1,4 +1,4 @@
-import copy
+from copy import deepcopy
```

Comment:
If we care about not mutating something we should probably care about not mutating it beyond the first level.

Reply:
Seems reasonable. Someone might have been thinking that `annotation.extra` wouldn't contain any nested mutable objects. And they might have been right as far as I know. But `deepcopy` doesn't have to rely on that assumption and I can't imagine it doing any _harm_

Reply:
Yeah, I don't think it's actually going to happen. But why not bring out the big guns?